src/container/BooksList.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import { connect } from 'react-redux';
+import Book from '../components/Book';
+
+const BooksList = ({ bookId, title, category }) => (
+  <table>
+    <tr>
+      <td>BookId</td>
+      <td>Title</td>
+      <td>Category</td>
+      <Book bookId={bookId} title={title} category={category} />
+    </tr>
+  </table>
+);
+
+BooksList.defaultProps = {
+  bookId: [],
+};
+
+const mapStateToProps = (state) => ({
+  books: state.books,
```

Comment:
- [ ] It would be better if we address the previous comment by the previous code reviewers. Now that we have access to the `books` array, through the `mapStateToProps`. We can now loop through the array and display the books as required